examples/tutorials/Introduction_to_ADCNet.ipynb

```diff
@@ -0,0 +1,1575 @@
+{
```

Comment:
<p>The model is fully pytorch style and not deepchem-ized at all. Please deepchem-ize the model so it is easier to integrate with the rest of deepchem</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4478/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='77'/>